browser/src/map/handler/Map.Feedback.js

```diff
@@ -84,15 +97,17 @@ L.Map.Feedback = L.Handler.extend({
 		}
 		else if (data == 'feedback-never') {
 			window.localStorage.setItem('WSDFeedbackEnabled', 'false');
+			window.localStorage.removeItem('WSDFeedbackCount');
 			this._iframeDialog.remove();
 		} else if (data == 'feedback-later') {
 			var currentDate = new Date();
 			this._iframeDialog.remove();
-			this._map.options.feedbackTimeout = 86400000;
-			window.localStorage.setItem('WSDFeedbackLaterDate', currentDate.toDateString());
-			setTimeout(L.bind(this.onFeedback, this), this._map.options.feedbackTimeout);
+			currentDate.setDate(currentDate.getDate() + 2);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate says getDate() returns the day of the month; so once it is set at the end of the month (ie. it becomes eg. 32), the if (... && currentDate > laterDate) above will never be true.

Would be better to use days since the start of epoch; like https://stackoverflow.com/questions/12739171/javascript-epoch-time-in-days .

Also I'd prefer to store the day of pressing the "Later" button, and check like "currentDay > laterButtonPressedDay + 2" so that it is easy to update the 2 to something else should it be too early to check again.